src/classes/ApexDataTypes.cls

```diff
@@ -0,0 +1,209 @@
+     *
+     *   @args           :   
+     *
+     *   @return         :   void
+     *
+     *   @revision Log   :   V1.1 - Created  - 06/24/2022 - Mrinal Das
+     *		          
+     * 
+     **/
+    
+    public Static void primitiveTypes() {
+        Boolean isWinner = true;
+        Double d=3.14159;
+        ID id='00300000003T2PGAA0';
+        Integer i = 1;
+        Long l = 21475698452L;
+        String s = 'The quick brown fox jumped over the lazy dog.';
+    }
+
+    /**
+     * 
+     *   @description    :  Practising Primitive Data Types
+     *
+     *   @args           :   
+     *
+     *   @return         :   void
+     *
+     *   @revision Log   :   V1.1 - Created  - 06/24/2022 - Mrinal Das
+     *		          
+     * 
+     **/
+    
+    //Date Class
+    public Static void addingDates(){
+        date myDate = date.newInstance(1990, 11, 21);
+        
+        //Adding Days
+    	date newDays = myDate.addDays(2);
+        System.debug('Adding Two Days From (1990, 11, 17)' + newDays);
+        
+        //adding Months
+        date newMonths = myDate.addMonths(3);
+        date expectedDate = date.newInstance(1991, 2, 21);
+        system.assertEquals(expectedDate, newMonths);
+        System.debug('Adding Three Months From (1990, 11, 17)' + newMonths);
+        
+        //adding Years
+        date newYears = myDate.addYears(2);
+        System.debug('Adding Three Months From (1990, 11, 17)' + newYears);
+        
+        //check day
+        Integer day = myDate.day();
+        system.assertEquals(21,day);
+        system.debug(day);
+        
+        //valueOf(stringDate)
+        string year = '2008';
+        string month = '10';
+        string days = '5';
+        string hour = '12';
+        string minute = '20';
+        string second = '20';
+        string stringDate = year + '-' + month
+         + '-' + days + ' ' + hour + ':' + 
+        minute + ':' + second;
+        
+        Date myDates = date.valueOf(stringDate);
+        system.debug(myDates);
+        
+    }
+
+    /**
+     * 
+     *   @description    :    Datetime Methods
+     *
+     *   @args           :   
+     *
+     *   @return         :   void
+     *
+     *   @revision Log   :   V1.1 - Created  - 06/24/2022 - Mrinal Das
+     *		          
+     * 
+     **/
+    
+    public Static void dateTimeMethod(){
+        //addDays
+        Datetime myDateTime = Datetime.newInstance(1960, 2, 17);
+        Datetime newDateTime = myDateTime.addDays(2);
+        Datetime expected = Datetime.newInstance(1960, 2, 19);
+        System.assertEquals(expected, newDateTime);
+    }
+
+    /**
+     * 
+     *   @description    :   Different String Methods
+     *
+     *   @args           :   
+     *
+     *   @return         :   void
+     *
+     *   @revision Log   :   V1.1 - Created  - 06/24/2022 - Mrinal Das
+     *		          
+     * 
+     **/
+
+    public Static void stringMethod() {
+        String s = 'Hello Maximillian';
```

Comment:
for string, you can write the Abbreviation str instate of s
str1, str2 maakes more sense than s